CRM/PivotReport/AbstractData.php

```diff
@@ -0,0 +1,526 @@
+<?php
+
+/**
+ * Provides a functionality to prepare entity data for Pivot Table.
+ */
+abstract class CRM_PivotReport_AbstractData implements CRM_PivotReport_DataInterface {
+  const ROWS_API_LIMIT = 1000;
+  const ROWS_PAGINATED_LIMIT = 10000;
+  const ROWS_MULTIVALUES_LIMIT = 1000;
+  const ROWS_RETURN_LIMIT = 10000;
```

Comment:
Can you please document what each of these constants represents and/or what they are used for?